### DISCUSSION TITLE
Document `<ins>` for underline formatting?

### DISCUSSION THREAD

**Discussion by johnthagen** [squidfunk/mkdocs-material#7091] [Ideas]
I was looking for how to <ins>underline</ins> text within a Material for Mkdocs project. I couldn't find it in either

- https://squidfunk.github.io/mkdocs-material/reference/formatting/

or the base Mkdocs docs

- https://www.mkdocs.org/

Finally, I found: https://stackoverflow.com/a/47354053 and this works (at least as of the latest Material for Mkdocs + Chrome).

I was wondering:

1. Is this correct/officially supported?
2. Since <ins>underline</ins> is somewhat basic/common, should the Material for Mkdocs docs call this out to help new users kind it more easily?

**Comment by johnthagen** [comment]
Nevermind, upon closer inspection I see: https://squidfunk.github.io/mkdocs-material/reference/formatting/?h=keys#highlighting-text

Perhaps what would be nice is it the string `underline` was somewhere in this docs section to improve search results.

**Reply by squidfunk** [reply]
I agree, it would indeed be a good addition to the documentation! We could add "(highlight)", "(underline)" and "(strikethrough)" after  "marked", "inserted" and "deleted" bullet points – could I maybe ask you to submit a PR?

**Reply by johnthagen** [reply]
@squidfunk

- #7092

**Comment by johnthagen** [comment]
Fixed in

- #7092
